### PR TITLE
Full website

### PR DESCRIPTION
**Pull request by JauJau088**
### Hi, in this PR, I added the following features:
- [x] Add 3 pages: Home, Calculator, and Quote
- [x] Use React basic styling
- [x] Use [React Router](https://reactrouter.com/) for the routing and navigation

A code review is much appreciated.

Thank you 🙏,
Jau

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,7 +1,22 @@
...
+import Quotes from './pages/Quotes';
+import Error from './pages/Error';
 
-const App = () => (<Calculator />);
+const App = () => (
+  <Router>
+    <Navbar />
+    <Routes>
+      <Route path="/" element={<Home />} exact />
+      <Route path="/calculator" element={<Calculator />} exact />
```
Comment (Mihndim2020) [inline src/App.js:14]: - [ ] As it is a [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_full_website.md) for this project, " `The general layout should match these wireframes:` ", please kindly adjust the layout of your pages and alignment of it's content to match the wireframe. The `Calculator page` is missing some text content (`Let's do some math!`), and the calculator itself occupies about 2/3 of the page and is left aligned. Please kindly fix that as it is a required skill for a front-end dev to stick to the design as much as  possible. Checkout the [wireframes](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_full_website.md).

**Review by Mihndim2020 — CHANGES_REQUESTED** [review]
### Hi @JauJau088, 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Descriptive Readme file
:heavy_check_mark: Great job with UI

There are some issues that you still need to work on to get this project approved, but you are almost there!
ALMOST-MINION

# Status: Requires Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 1 above)
Comment (JauJau088) [reply src/App.js:14]: Hi,

Thank you for the well-described feedback!

Updated now.

**Review by Kingstalux — APPROVED** [review]
Hi @JauJau088,
# Status: Approved ✔️ ✔️ ✔️ 
 
68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f6d43496a436773336e575157664a5a7650412f67697068792d646f776e73697a65642d6c617267652e676966


Good job implementing the required changes 👍 Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by JauJau088** [comment]
#### Thank you all for the reviews. Really appreciate them! 🙏 

I am merging this now.

Cheers, 🥂 
Jau

**Pull request merged**